TombEngine/Game/items.cpp

```diff
@@ -42,16 +42,29 @@ constexpr auto ITEM_DEATH_TIMEOUT = 4 * FPS;
 
 BoundingBox ItemInfo::GetAabb() const
 {
-	return Geometry::GetBoundingBox(GetObb());
+	return Geometry::GetAabb(GetObb());
 }
 
 BoundingOrientedBox ItemInfo::GetObb() const
 {
 	auto frameData = GetFrameInterpData(*this);
 	if (frameData.Alpha == 0.0f)
-		return frameData.FramePtr0->BoundingBox.ToBoundingOrientedBox(Pose);
+	{
```

Comment:
This yields incorrect results compared to `return frameData.Keyframe0.BoundingBox.ToBoundingOrientedBox(Pose)`.